maven/versions-rules.xml

```diff
@@ -11,6 +11,25 @@
     <ignoreVersion type="regex">(?i).*RC(?:-?\d+)?</ignoreVersion>
     <ignoreVersion type="regex">(?i).*CR(?:-?\d+)?</ignoreVersion>
     <ignoreVersion type="regex">(?i).*M(?:-?\d+)?</ignoreVersion>
+
+    <!-- To be reviewed -->
+    <ignoreVersion type="regex">.*.alpha.*</ignoreVersion>
+    <ignoreVersion type="regex">.*.beta</ignoreVersion>
+    <ignoreVersion type="regex">.*-b01</ignoreVersion>
+    <ignoreVersion type="regex">.*.pr[0-9]</ignoreVersion>
+    <ignoreVersion type="regex">.*-eap-.*</ignoreVersion>
+    <ignoreVersion type="regex">.*-RC[0-9].*</ignoreVersion>
+    <ignoreVersion type="regex">.*.RC[0-9].*</ignoreVersion>
+    <ignoreVersion type="regex">.*-rc</ignoreVersion>
+    <ignoreVersion type="regex">.*.M[0-9].*</ignoreVersion>
+    <ignoreVersion type="regex">.*-SNAP[0-9]+</ignoreVersion>
+    <ignoreVersion type="regex">.*-rc[0-9]+</ignoreVersion>
+    <ignoreVersion type="regex">.*-rc-[0-9]+</ignoreVersion>
+    <ignoreVersion type="regex">.*-beta[0-9]+</ignoreVersion>
+    <ignoreVersion type="regex">.*-beta-[0-9]+</ignoreVersion>
+    <ignoreVersion type="regex">.*.kotlin12+</ignoreVersion>     
```

Comment:
Disse to reglene (kotlin12 + native-mt) høres ut som noe som ikke hører hjemme i et generisk oppsett som dette

Reply:
Tenker du at all kodespråk-spesifikke ting bør fjernes? jre-delen også, da?